en/flight_controller/pixhawk_series.md

```diff
@@ -89,4 +89,7 @@ Product names/brands can also be trademarked. Trademarked names may not be used
 
 ### LEDs
 
-All *Pixhawk-series* flight controllers have an RGB LED to indicate the current status of the vehicle. To interpret the LEDs see: [LED Meanings](../flying/led_meanings.md).
+All *Pixhawk-series* flight controllers have an superbright RGB *UI LED* to indicate the current *readiness to fly* status of the vehicle. 
```

Comment:
> All *Pixhawk-series* flight controllers have an superbright RGB *UI LED* to indicate the current *readiness to fly* status of the vehicle. 

All *Pixhawk-series* flight controllers support user facing RGB *UI LED* to indicate the current *readiness to fly* status of the vehicle. This is typically a superbright I2C peripheral. 
  
   
Some are not on board. I..E are mounted on the GPS or a separate box. FmuV4 does not have one on board.